tests/python/test_multidevice.py

```diff
@@ -1623,6 +1623,7 @@ def test_transformer_backward(multidevice_test, benchmark):
     mha_linear0_weight = torch.testing.make_tensor(
         d, e * 3 // d, e, dtype=torch.bfloat16, device="cpu"
     )
+    philox_seed, philox_offset = get_sdpa_rng_tensors()
```

Comment:
```suggestion
    sdpa_philox_seed, sdpa_philox_offset = get_sdpa_rng_tensors()
```
to distinguish them from other random seeds and offsets e.g. L1628-1629 and L1633-1634